### PR TITLE
[NUI] Add FloatGraphicsTypeConverter and support converter in PixelSize in Text.

### PR DESCRIPTION
**Pull request by everLEEst**
### Description of Change ###
[NUI] Add FloatGraphicsTypeConverter and support converter in PixelSize in Text.

### API Changes ###
none

### CODE REVIEW

File: src/Tizen.NUI/src/public/XamlBinding/FloatGraphicsTypeConverter.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright(c) 2021 Samsung Electronics Co., Ltd.
```
Comment (Seoyeon2Kim) [inline src/Tizen.NUI/src/public/XamlBinding/FloatGraphicsTypeConverter.cs:2]: 2022!

File: src/Tizen.NUI/src/public/XamlBinding/FloatGraphicsTypeConverter.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+using System.Reflection;
+using System.Globalization;
+
+using Tizen.NUI;
+using Tizen.NUI.Xaml;
+using System.ComponentModel;
+
+namespace Tizen.NUI.Binding
+{
+    /// This will be public opened in tizen_5.0 after ACR done. Before ACR, need to be hidden as inhouse API.
```
Comment (Seoyeon2Kim) [inline src/Tizen.NUI/src/public/XamlBinding/FloatGraphicsTypeConverter.cs:29]: Could you update not to use `tizen_5.0` anymore?

**Review by Seoyeon2Kim — COMMENTED** [review]
Minor comments!

**Comment by dongsug-song** [comment]
@everLEEst 
Hello, could you write the description? and also check Seoyeon2Kim's comment?
thank you.

**Review by jaehyun0cho — APPROVED** [review]
LGTM

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Pull request merged**